active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,393 @@
+| PagerControlPageChangedEvent | Event that is fired when the end user selects a button from the number panel, any of the 4 directional buttons, or selects an option from the combobox or enters a number in the numberbox. This event will return the index number that the end user selected. If the end user enters a number greter than the number of pages this event will return the last page. If the end user enters a number less than 1 the event will return 1. | N/A |
+
+# Design Components and Functionality
+
+ | Component |  Notes |
+|:---:|:---|
+| LastButton | * Button displaying text and/or glyph indicating that the user may navigate to the last index. <br> * Automatically disabled when at last index. <br> * Can be set to not be visible when at the last index.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760392-3a0d6980-fc79-11e8-8d96-a811cee1c759.png) |
+| FirstButton | * Button displaying text and/or glyph indicating that the user may navigate to the first index. <br> * Automatically disabled when at first index. <br> * Can be set to not be visible when at the first index.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760409-485b8580-fc79-11e8-9545-a580f6016f6c.png) |
+| NextButton | * Button displaying text and/or glyph indicating that the user may navigate to the next index. <br> * Automatically disabled when at last index. <br> * Can be set to not be visible when at the last index. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760426-54474780-fc79-11e8-9446-ee2886941a24.png) |
+| PreviousButton | * Button displaying text and/or glyph indicating that the user may navigate to the previous index. <br> * Automatically disabled when at first index. <br> * Can be set to not be visible when at the first index.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760443-62956380-fc79-11e8-8f93-562a92fde666.png) |
+| Ellipsis | * Button, reading "...", used before or after the first/last index to indicate an accessible but omitted range of indexes. <br> * Visibility of the first/last index can be disabled. <br> <br> * Only visible when using button panel as the display mode.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760513-85277c80-fc79-11e8-926e-3453ea29b0a3.png) |
+| PrefixText | * Text displayed before the ComboBox or NumberBox indexing component. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760540-93759880-fc79-11e8-9c4f-3c3222b95a13.png) |
+| SuffixText | * When a total number of indices (N) is given, this suffix string will appear after the ComboBox indexing component or NumberBox component and read "of N". Localization will put "N" where it should be in a given language. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760557-9a9ca680-fc79-11e8-943f-3425d993fdeb.png) |
+
+# API Details
+
+```c++
+enum PagerDisplayMode
```

Comment:
If the control is named PagerControl, would it be better to name this PagerControlButtonVisibility?

The event args are also named PagerControl... so maybe we should do this here too?

Reply:
@MikeHillberg do you have an opinion on if this should have the word control the API name?

Reply:
I think it makes sense to call it <ControlName>DisplayMode similar to others like NavigationViewDisplayMode. Property names don't need it since we always use it in the context of the control. So the enum would be PagerControlDisplayMode and the property name would be DisplayMode.

Reply:
I would leave off the "Mode" though; that word always feels good to add to API names, but it doesn't really provide any value.

Reply:
I'm confused, @MikeHillberg are you suggesting removing the 'Mode' so it would be 'PagerControlDisplay' and 'Display' ? That sounds weird to me and would be inconsistent with what we already have in other places like NavigationView.

Reply:
You're right, lots of precedent for ...DisplayMode. Sounds good to me too.

Reply:
Updated to add control to visibility behavior and display mode.